### DISCUSSION TITLE
[scam Alert] pcoin (puma coin global)

### DISCUSSION THREAD

**Discussion by micro73** [Blockunity/WarningList#35] [General]
pcoin (puma coin global)

ادرس سایت : https://pcocrypto.com/.

هشدار کلاهبرداری و اسکم 

توضیح سایت: 
ما یک پلت فرم ساخته ایمبرای خرید و فروش سهام.
PCCoin Global یک پلت فرم برای آینده تامین مالی است که به بلاک چین جدید سهام قدرت می دهد
در حالی که راه‌حل‌های موجود تنها برای حل یک مشکل در یک زمان ارائه می‌شوند، تیم ما آماده است تا یک محصول ایمن، مفید و با استفاده آسان بر اساس بلاک‌چین خصوصی بسازد. این شامل یکپارچه‌سازی آسان پرداخت‌های ارز دیجیتال و حتی یک سیستم داوری دیجیتال خواهد بود.
در پایان، هدف ما این است که همه شرکت‌ها، کارمندان و دارایی‌های تجاری را در یک اکوسیستم بلاک چین یکپارچه ادغام کنیم، که تجارت را واقعاً کارآمد، شفاف و قابل اعتماد خواهد کرد.

بررسی های صورت گرفته ::

ابتدا اینکه تیم سازنده این پلتفرم کاملا نامشخص و هویتی ندارند 

به محض ورود به سایت trend micro check هشدار scam page میدهد
![2022-08-19_11-49-48](https://user-images.githubusercontent.com/103103225/185568980-c752240c-dee3-418d-be96-43344c2796b0.jpg)

بررسی اسکم ادوایرز سایت و ترجمه ماشینی ::
صاحب وب سایت از سرویسی برای پنهان کردن هویت خود در WHOIS استفاده می کند
طبق گفته Tranco این سایت دارای رتبه الکسا پایینی است
این وب سایت ممکن است خدمات ارزهای دیجیتال پرخطر ارائه دهد
این فروشگاه اینترنتی روش های پرداخت ناشناس را ارائه می دهد.
این وب سایت به عنوان یک کلاهبرداری احتمالی در Scamadviser گزارش شده است
ما ارجاعاتی به تلاش‌های بازاریابی چندسطحی را شناسایی کردیم که اغلب کلاهبرداری هستند
این وب سایت عمدتاً نقدهای منفی دریافت کرده است
این وب سایت (بسیار) جوان است.
این وب سایت در 30 روز گذشته توسط DNSFilter به عنوان تهدید علامت گذاری شده است
![2022-08-19_12-11-24](https://user-images.githubusercontent.com/103103225/185569252-2955ba4e-4e9d-4304-890e-0c20b2b910a1.jpg)
![2022-08-19_12-12-41](https://user-images.githubusercontent.com/103103225/185569357-521ebaaf-597d-4e43-ab89-8ca2f78ba85b.jpg)

اطلاعات whois سایت 👍 
اطلاعات کاملا پروتکت شده هستند و از پرووایدری استفاده میکنند که در بررسی های قبلی مشخص شد اکثر اسکمر ها و کلاهبرداران از این پرووایدر استفاده میکنند
![2022-08-19_11-52-41](https://user-images.githubusercontent.com/103103225/185569601-05f2bea1-3379-41ee-be0b-4b1fd73cf363.jpg)

شبکه های اجتماعی این سایت توییتر و تلگرام و اینستاگرام کاملا جدید و بدون کاربر و بدون اطلاعات هستند 
![2022-08-19_11-56-23](https://user-images.githubusercontent.com/103103225/185569952-adddf290-31dc-4184-aa2d-23845792ce5b.jpg)
![2022-08-19_11-56-40](https://user-images.githubusercontent.com/103103225/185569957-3fffaae3-e769-4107-8793-bd86b9037bf8.jpg)
![2022-08-19_12-07-27](https://user-images.githubusercontent.com/103103225/185569961-aa67fcd6-f5b6-4cde-bdb7-df833f16fde9.jpg)
![2022-08-19_12-07-58](https://user-images.githubusercontent.com/103103225/185569975-e4921652-95b1-4696-8333-e3f5a892c116.jpg)

ورودی های سایت تماما کاربران ایرانی هستند 

![2022-08-19_11-54-19](https://user-images.githubusercontent.com/103103225/185570027-5b602ae1-bfae-4e30-bf4f-10c7ed5ad600.jpg)

**Comment by micro73** [comment]
در سایت فقط چندین جمله درباره نحوه فعالیت نوشته شده و هیچ ادرس کانترکتی و اطلاعات افراد گرداننده پروژه موجود نیست در واقع فقط حرف زده شده است

**Reply by OMid7313** [reply]
اطلاعات افراد گرداننده بیت کوین موجود هست اگه موجود داری بیار دنیا مشتاقه ببینش 😂

**Comment by Blockunity** [comment]
ادعا ثبت شرکت در انگلیس
[13851933_newinc_2022-01-17.pdf](https://github.com/Blockunity/WarningList/files/9381645/13851933_newinc_2022-01-17.pdf)

ادعای داشتن مزرعه که بدون سند هست.
ادعای داشتن بلاکچین اختصاصی که هیچ اطلاعات تکنیکی ازش موجود نیست حتی اکانت گیتهابی وجود ندارد
صرافی معرفی شده:
https://novintex.com/
صرافی خدمات دهنده هست.


[pcoin-whiepaper-v0.1.pdf](https://github.com/Blockunity/WarningList/files/9381642/pcoin-whiepaper-v0.1.pdf)
در وایت پیپر تمامی ادعا ها بی اساس بوده و درحد نوشتار هست. توزیع توکن کاملا خلاف وایت پیپر هست.

https://tronscan.org/#/token20/TYnBXH9TsuzY8rviUN5EKytKf7D99z3gci

کانترکت این پلفرم بر بستر ترون هست.
![Screenshot 2022-08-19 121306](https://user-images.githubusercontent.com/72666334/185597289-609c24a6-9451-4a58-b17a-2a646794ce18.png)

نوع توزیع ارز بسیار شبیه به BRG هست و 90 درصد در دست 5 ولت اول هست.

کانترکت این پروژه توسط zeppelin راه اندازی شده.
خطرات:
فانکشن مینت که اجازه تولید بیش از مقدار مشخص شده در ماکس ساپلای هست 
فانکشن سوزاندن که صاحب پروژه اجازه سوزوندن دارد.
 /**
     * @dev Destoys `amount` tokens from `account`.`amount` is then deducted
     * from the caller's allowance.
     *
     * See {_burn} and {_approve}.
     */
    function _burnFrom(address account, uint256 amount) internal {
        _burn(account, amount);
        _approve(account, msg.sender, _allowances[account][msg.sender].sub(amount));

نوع هشدار: اسکم و راگ پول
از خرید این ارز و سرمایه گذاری در این سایت دوری کنید

**Comment by Blockunity** [comment]
https://youtu.be/S0e6I4ix_4c

ویدیو تهیه شده در یوتوب

**Comment by Blockunity** [comment]
جهت تکمیل اطلاعات سعی کردیم که پرزنت بشیم توسط افراد این پروژه.
صحبت های مختلفی گفته شد.
در پرزنت اول مدعی داشتن مزرعه بیتکوین بودند
در پرزنت دوم این ادعا رو رد کردند
بیشتر درباره پرزنت دوم صحبت میکنیم:
این پروژه طبق ادعای لیدر ها سرمایه گذاری در کارخانه و زمین داشته.
این املاک به نام فرد هست تاشرکت و این به این معنی هست که فرد صاحب اون سرمایه هست و نه شرکت.
این سرمایه ها به عنوان محل کسب درامد و پشتوانه مالی قرار داده شده!
ادعای دیگر که هیچ ربطی به اساس نامه کمپانی ندارد اجرا کردن DPOS هست که مختصر 
Delegate Proof of stake 
هست که عملا یک الگوریتم اجماع جدا از POS-POW مدعی شده در وایت پیپر هست.
مدعی هستند که بلاکچین راه اندازی شده و همزمان روی بلاکچین ترون قرار گرفته!!!
از نظر فنی این ادعا کاملا غلط و ناشیانه هست!
ادعای دیگه این هست که تیم برای اجرای پروژه از سرمایه سرمایه گذار ها استفاده میکنه!
این عملا کلاهبرداری هست.
در حالی که تمامی پروژه های بلاکچینی برای راه اندازی بلاکچین اختصاصی خودشان یا جذب سرمایه از شرکت های سرمایه گذار میکنند یا خود شرکت سرمایه گذاری میکند و همچنین تمام سورس های بلاکچین و تست نت و فعالیت داخل گیت هاب قرار میگیرد جهت بررسی.
ادعای راه اندازی یک صرافی غیر متمرکز دارند در کنار novintex 
همچنان این صرافی هیچ گونه مجوز فعالیتی ندارد.
یک تغییر در توزیع توکن ها صورت گرفته که از ۱۰ ولت به ۵۰ ولت تغییر کرده و بیش از ۹۰ درصد از ساپلای دست ۵۰ ولت هست.

ایراد فنی دیگر که تایید بر کلاهبردار بودن این پروژه میکند:
در مبحث POW یا POS شما در مقابل فعالیتی یک پاداشی از شبکه دریافت میکنید چه به عنوان ماینر در POW چه به عنوان ولیدیتور یا اعتبار سنج در POS.
زمانی که شما دارید در یک بلاکچین استخراج انجام میدید آن معدن که شامل بلاک ها هست یا یک ساپلای مشخص مثل بیتکوین دارد که ۲۱ میلیون عدد هست یا مثل اتریوم نا محدود هست، شما بابا فعالیت اون ارز رو تولید میکنید مثل یک معدن طلا که هرچه بیشتر بکنید بیشتر طلا استخراج میکنید و عملا هنوز اون معدن شامل طلا هست برای استخراج.
در پروژه پوما کوین که ادعای ماینینگ به واسطه استیک شما در پلتفرم خودشون رو دارند متاسفانه این ادعا کاملا غلط هست چون تمام این ارز استخراج شده ودر شبکه موجود هست.
عملا شما فعالیتی انجام نمیدید که به ما پاداشی تعلق بگیره پس عملا پروژه پول خود شما رو خورد خورد به شما پرداخت میکند و ترغب به سرمایه‌گذاری دوباره میکند.


همچنان این پروژه در دسته بندی اسکم هست.

**Comment by mjz10** [comment]
یکی از دوستای من 2000 دلار توی پروژه کلاهبرداری اوربیت نتورک گذاشته بود به منم پیشنهاد داد برم زیر مجموعه اش. من رفتم سایت اوربیت نتورک رو بررسی کردم دیدم سه تا ار ز nye و rkn  و snl داره که وقتی توی کوین مارکت کپ بررسیشون میکنی میفهمی چه فاجعه ای هستن. از تعداد فالور توییتر زیر هزارتا تا توزیع توکن هاشون توی سه تاولت و پنهان بودن سازنده سایت و حجم معاملات روزانه زیر 2000 دلار و لیست شدن فقط توی یه صرافی و کلی باگ دیگه. 
با دوستم با این سر دسته زیرمجموعه بگیرشون قرار گذاشتیم توی کافه سه تا داف اومده بودن مثلا منو تتحت تاثیر بذارن
موارد بالا رو گذاشتم جلوشون باورتون نمیشه اولین چیزی که گفت چی بود!! 
گفت من وقت ندارم بخوام جواب بدم ساعتی 100 دلار به ضررمه اگه شما جواب این سوالات رو میخواید باید 10 هزار دلار زمین متاورسی بخرید تا جواب بدم
منم گفتم عههه ینی اگه سوال نمیپرسیدم شما با همون 2000 دلار اوکی بودین؟ ینی نباید کلا سوال میپرسیدم ؟ وقت شما وقت هست وقت ما نه ؟ گفتم پروژه شما راسش اشکان اتشکار هست که الان توی ترکیه هست و پول مردم دستشه 
خلاصه جور شستمشون که حرفی برا گفتن نداشتن .
دیگه من شیک شکلاتی به حساب خودشون زدم به بدنو اومدم ولی حیف که نمیشه اینا رو تحویل قانون داد چون اساسا قانونی توی جمهوری اعدامی برای این کلاهبرداران وجود نداره.

**Reply by mjz10** [reply]
در تکیملش بگم اینا این پولو میگیرن که زمین توی پروژه اولتراورسیتی بخرن که ربطی به اوربیت نتورک نداره گویا و ادعاشون اینه قیمت زمیناشون زیر 2000 دلار نمیاد و حتما افزایش پیدا میکنه و این ینی دستکاری توی عرضه وتقاضا . خلاصه گول این افراد رو نخورید
